### ISSUE TITLE
Add a parentNode prop

### ISSUE THREAD

**Issue by philbot9** [RRutsche/react-parallax#25]
Hi there,

First, great work I really like this package.

In one of my projects, the Parallax component lives inside a div with a vertical overflow scroll. However, because the Parallax component attaches the 'scroll' event handler to the window, there is no parallax effect as I scroll the div.

I've created a dirty fix by adding a parentNode prop and attaching the DOM event handlers to it instead of the window. (https://github.com/philbot9/react-parallax/commit/888c0b14b17b3288cad945306a32cf5fc943667f)

Then it's used like this:
``` javascript
<Parallax parentNode={document.querySelector('#scrollable-div')}> ...
```

Would you be open to (properly) implementing a 'parentNode' prop?

**Comment by rrutsche** [comment]
Hi @philbot9 thanks for your feedback. Of course I can have look at your idea and check your example implementation.
I will give you a note.
- regards
R

**Comment by philbot9** [comment]
Thanks @RRutsche! It turns out my solution is causing some errors. I've come across this package, which solves a similar problem by using a parent function.

https://kyleamathews.github.io/react-headroom/

Form the docs:
"parent — provide a custom 'parent' element for scroll events. parent should be a function which resolves to the desired element."

So I think the solution would be similar.

**Comment by rrutsche** [comment]
Hi @philbot9!
I just published version [1.2.7](https://github.com/RRutsche/react-parallax/releases/tag/v1.2.7) on [npm](https://www.npmjs.com/package/react-parallax). I did not solve it with a function but with a simple HTMLElement prop. Let me hear if it works for you.

regards

**Comment by rrutsche** [comment]
@philbot9 I will close the issue as you did not have any complaints. Feel free to comment/open if you have any issue with the update.
